2025-04-01T14:48:25.020526631Z W0401 14:48:25.020151       1 feature_gate.go:354] Setting GA feature gate Topology=true. It will be removed in a future release.
2025-04-01T14:48:25.020661139Z I0401 14:48:25.020652       1 csi-provisioner.go:154] Version: v5.1.0
2025-04-01T14:48:25.020683629Z I0401 14:48:25.020676       1 csi-provisioner.go:177] Building kube configs for running in cluster...
2025-04-01T14:48:28.127365832Z I0401 14:48:28.127309       1 common.go:143] "Probing CSI driver for readiness"
2025-04-01T14:48:28.131261851Z I0401 14:48:28.131227       1 common.go:143] "Probing CSI driver for readiness"
2025-04-01T14:48:28.134516331Z I0401 14:48:28.134161       1 csi-provisioner.go:299] CSI driver supports PUBLISH_UNPUBLISH_VOLUME, watching VolumeAttachments
2025-04-01T14:48:28.135079831Z I0401 14:48:28.135044       1 leaderelection.go:254] attempting to acquire leader lease kube-system/cinder-csi-openstack-org...
2025-04-01T14:48:28.156729463Z I0401 14:48:28.156683       1 leaderelection.go:268] successfully acquired lease kube-system/cinder-csi-openstack-org
2025-04-01T14:48:28.258456376Z I0401 14:48:28.258365       1 clone_controller.go:66] Starting CloningProtection controller
2025-04-01T14:48:28.258641184Z I0401 14:48:28.258597       1 clone_controller.go:82] Started CloningProtection controller
2025-04-01T14:48:28.258715873Z I0401 14:48:28.258463       1 controller.go:824] "Starting provisioner controller" component="cinder.csi.openstack.org_csi-cinder-controllerplugin-5847b67f7b-fdzk8_44556425-4f0f-49d2-ada9-754e2bd9e6d9"
2025-04-01T14:48:28.258825821Z I0401 14:48:28.258298       1 volume_store.go:98] "Starting save volume queue"
2025-04-01T14:48:28.360120481Z I0401 14:48:28.359665       1 controller.go:873] "Started provisioner controller" component="cinder.csi.openstack.org_csi-cinder-controllerplugin-5847b67f7b-fdzk8_44556425-4f0f-49d2-ada9-754e2bd9e6d9"
2025-04-01T14:48:28.360558794Z I0401 14:48:28.360525       1 event.go:389] "Event occurred" object="default/csi-pvc-cinderplugin" fieldPath="" kind="PersistentVolumeClaim" apiVersion="v1" type="Normal" reason="Provisioning" message="External provisioner is provisioning volume for claim \"default/csi-pvc-cinderplugin\""
2025-04-01T14:48:30.139313543Z I0401 14:48:30.139238       1 event.go:389] "Event occurred" object="default/csi-pvc-cinderplugin" fieldPath="" kind="PersistentVolumeClaim" apiVersion="v1" type="Normal" reason="ProvisioningSucceeded" message="Successfully provisioned volume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea"
2025-04-02T07:42:42.071827625Z E0402 07:42:42.071770       1 controller.go:1558] "Volume deletion failed" err="persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1" PV="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea"
2025-04-02T07:42:42.071918394Z I0402 07:42:42.071908       1 controller.go:1007] "Retrying syncing volume" key="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea" failures=0
2025-04-02T07:42:42.071953993Z E0402 07:42:42.071944       1 controller.go:1025] "Unhandled Error" err="error syncing volume \"pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea\": persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1" logger="UnhandledError"
2025-04-02T07:42:42.071996182Z I0402 07:42:42.071987       1 event.go:389] "Event occurred" object="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea" fieldPath="" kind="PersistentVolume" apiVersion="v1" type="Warning" reason="VolumeFailedDelete" message="persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1"
2025-04-02T07:42:43.072275804Z E0402 07:42:43.072191       1 controller.go:1558] "Volume deletion failed" err="persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1" PV="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea"
2025-04-02T07:42:43.072275804Z I0402 07:42:43.072250       1 controller.go:1007] "Retrying syncing volume" key="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea" failures=1
2025-04-02T07:42:43.072370812Z E0402 07:42:43.072276       1 controller.go:1025] "Unhandled Error" err="error syncing volume \"pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea\": persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1" logger="UnhandledError"
2025-04-02T07:42:43.072473130Z I0402 07:42:43.072457       1 event.go:389] "Event occurred" object="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea" fieldPath="" kind="PersistentVolume" apiVersion="v1" type="Warning" reason="VolumeFailedDelete" message="persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1"
2025-04-02T07:42:45.072673931Z E0402 07:42:45.072557       1 controller.go:1558] "Volume deletion failed" err="persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1" PV="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea"
2025-04-02T07:42:45.072673931Z I0402 07:42:45.072642       1 controller.go:1007] "Retrying syncing volume" key="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea" failures=2
2025-04-02T07:42:45.072737601Z E0402 07:42:45.072668       1 controller.go:1025] "Unhandled Error" err="error syncing volume \"pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea\": persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1" logger="UnhandledError"
2025-04-02T07:42:45.072840670Z I0402 07:42:45.072814       1 event.go:389] "Event occurred" object="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea" fieldPath="" kind="PersistentVolume" apiVersion="v1" type="Warning" reason="VolumeFailedDelete" message="persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1"
2025-04-02T07:42:49.073484836Z E0402 07:42:49.073409       1 controller.go:1558] "Volume deletion failed" err="persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1" PV="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea"
2025-04-02T07:42:49.073484836Z I0402 07:42:49.073470       1 controller.go:1007] "Retrying syncing volume" key="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea" failures=3
2025-04-02T07:42:49.073550625Z E0402 07:42:49.073498       1 controller.go:1025] "Unhandled Error" err="error syncing volume \"pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea\": persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1" logger="UnhandledError"
2025-04-02T07:42:49.073567395Z I0402 07:42:49.073553       1 event.go:389] "Event occurred" object="pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea" fieldPath="" kind="PersistentVolume" apiVersion="v1" type="Warning" reason="VolumeFailedDelete" message="persistentvolume pvc-a8be0808-3521-45d1-baed-6c1c3b9b4fea is still attached to node host-10-13-2-1"
